mapie/control_risk/ltt.py

```diff
@@ -0,0 +1,120 @@
+import warnings
+
+from typing import Tuple, List, Optional, Any
+from mapie._typing import NDArray, ArrayLike
+
+import numpy as np
+
+from .p_values import compute_hoefdding_bentkus_p_value
+
+
+def _ltt_procedure(
+    r_hat: NDArray,
+    alpha_np: NDArray,
+    delta: Optional[float],
+    n_obs: int
+) -> Tuple[List[List[Any]], NDArray]:
+    """
+    Apply the Learn-Then-Test procedure for risk control.
+    This procedure is called in multi-label-classification
+    if ``metric=precision``.
+    Note that we will do a multiple test for ``r_hat`` that are
+    less than level ``alpha_np``.
+
+    Procedure:
+        - Calculate p-values for each lambdas descretized
+        - Apply a fwer algorithm, here Bonferonni correction
+        - Return the index lambdas that give you the control
+        at alpha level
+
+    Parameters
+    ----------
+    r_hat: NDArray of shape (n_lambdas, ).
+        Empirical risk of metric_control with respect
+        to the lambdas.
```

Comment:
```suggestion
    r_hat: NDArray of shape (n_lambdas, ).
        Empirical risk with respect to the lambdas.
```

Reply:
lambdas are thresholds in this case, it is right to speak about parameters, because they are for ltt procedure. But In this case they are also thresholds for decision making.

Reply:
Agree to not restrict to thresholds. I suggest at least to talk about their impact on decision-making in multi label classification.

Reply:
Done